scripts/ecdsa/eigen/update_quorum.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# -x echos all lines for debug
+# set -x
+
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+# shellcheck source=../../helper.sh
+# shellcheck disable=SC1091
+source "$SCRIPT_DIR/../../helper.sh"
+
+# Parse command line arguments in key=value format
+parse_args "$@"
+
+# Check required parameters with defaults
+check_param "DEPLOY_ENV" "${DEPLOY_ENV:-LOCAL}"
+check_param "WAVS_SERVICE_MANAGER_ADDRESS" "${WAVS_SERVICE_MANAGER_ADDRESS:-$(jq -r '.addresses.WavsServiceManager' "$HOME/.nodes/avs_deploy.json")}"
+check_param "QUORUM_NUMERATOR" "${QUORUM_NUMERATOR:-1}"
```

Comment:
I don't think `${QUORUM_NUMERATOR:-1}` does what you want. It hardcodes 1 as default, rather than defaulting to `$1`. See my foo.sh example:

```bash
#!/bin/sh

echo ${FOO:-1}
```
Result:
```
$ FOO=3 ./foo.sh
3
$ ./foo.sh
1
$ ./foo.sh 33
1
```

Reply:
Try this one:

```bash
#!/bin/sh

echo ${FOO:-$1}
```

Result:

```
$ FOO=3 ./foo.sh
3
$ ./foo.sh 33
33
./foo.sh

```

Which will accept either the env variable set (directly or passed as kv pairs via your parse_args), or the number passed directly as a cli arg. And if neither is passed, it will error.

Reply:
fixed